tasks/quality_gates.py

```diff
@@ -179,8 +179,10 @@ def parse_and_trigger_gates(ctx, config_path: str = GATE_CONFIG_PATH) -> list[St
                 }
             )
         except InfraError as e:
+            print("!" * 50)
             print(f"Gate {gate.config.gate_name} flaked ! (InfraError)\n Restarting the job...")
             traceback.print_exception(e)
+            print("!" * 50)
```

Comment:
Side note: you may also achieve highlighting with
```suggestion
            print(color_message(f"Gate {gate.config.gate_name} flaked ! (InfraError)\n Restarting the job...", "red"))
            for line in traceback.format_exception(e):
                print(color_message(line, "red"))
```